pyproject.toml

```
@@ -21,21 +21,12 @@ packages = [{ include = "qibolab", from = "src" }]
 include = ["*.out", "*.yml"]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.13"
+python = ">=3.10,<3.14"
 qibo = "^0.2.16"
-numpy = "^1.26.4"
+numpy = ">=1.26.4"
```

```suggestion
numpy = ">=1.26.4,<3"
```
Just not to leave the upper bound completely free